### DISCUSSION TITLE
Ability to Create Custom Prompts Without Internet Search

### DISCUSSION THREAD

**Discussion by guilhermeaddr** [qunash/chatgpt-advanced#53] [Ideas]
Dear WebChatGPT Team,

I am a user of the WebChatGPT browser extension and I love the Prompt Editor feature. However, I would like to create custom prompts without the requirement of an internet search.

I would greatly appreciate it if this feature could be added to the extension. It would allow me to create more versatile and customized prompts to better fit my needs.

Thank you for your hard work and dedication in creating such a great tool. I look forward to hearing back from you.

**Comment by qunash** [comment]
Hi, thanks for the suggestion!

The Prompt Editor is designed to require the {web_results} placeholder in prompts, since the primary function of the extension is to assist with web search. However, I can see how it would be useful to create prompts without the need for search results. I will think about making this change in the future.
